doc/source/whatsnew/v1.3.4.rst

```diff
@@ -34,14 +34,16 @@ Bug fixes
 - Fixed bug in :meth:`.GroupBy.mean` with datetimelike values including ``NaT`` values returning incorrect results (:issue:`43132`)
 - Fixed bug in :meth:`Series.aggregate` not passing the first ``args`` to the user supplied ``func`` in certain cases (:issue:`43357`)
 
+
 .. ---------------------------------------------------------------------------
 
 .. _whatsnew_134.other:
 
 Other
 ~~~~~
--
--
+- Improved performance of :meth:`pandas.read_csv` with ``memory_map=True`` when file encoding is UTF-8 (:issue:`43787`)
```

Comment:
there is a peformance section, pls move there

Reply:
Sorry, I didn't find this section; I moved it now.
I am still working on asv bench and asv perf test, hope to have it soon.